README.md

```diff
@@ -28,15 +28,15 @@ dist/omniSearch.js
 Run this to update files
 
 ```
-yarn
-yarn dev
+npm install
+npm run dev
 ```
 
 ### Coding standards
 
 Run prettier via the following command:
 
 ```shell
-    docker run -it --volume ${PWD}:/app --rm node:20 yarn --cwd /app install
-    docker run -it --volume ${PWD}:/app --rm node:20 yarn --cwd /app prettier src --fix --write
+    docker run -it --volume ${PWD}:/app --rm node:20 npm --prefix /app install
+    docker run -it --volume ${PWD}:/app --rm node:20 npm --prefix /app prettier src --fix --write
```

Comment:
```suggestion
    docker compose run --rm php npm --prefix /app install
    docker compose run --rm php npm --prefix /app prettier src --fix --write
```

Reply:
This does not work. but the thing I wrote doesn't work either.